src/Acorn/Assets/Asset.php

```diff
@@ -57,9 +57,20 @@ public function contents(): string
         if (! $this->exists()) {
             return false;
         }
+
         return file_get_contents($this->path());
     }
 
+    /** {@inheritdoc} */
+    public function get(): array
+    {
+        if (! $this->exists()) {
+            return false;
+        }
+
+        return require_once($this->path());
```

Comment:
RRRREEEEEEEEEEEEEEEEEEEE

`require_once $this>path();`

But I also think we should use `include`.